vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_wallet.cc

```diff
@@ -166,8 +167,8 @@ void UpholdWallet::OnCreateCard(
 ledger::ExternalWalletPtr UpholdWallet::SetStatus(
     const User& user,
     ledger::ExternalWalletPtr wallet) {
-  switch (wallet->status) {
-    case ledger::WalletStatus::CONNECTED: {
+  switch (static_cast<int>(wallet->status)) {
```

Comment:
Just curious, why cast here? if wallet->status is already of ledger::WalletStatus enum type.

Reply:
because mojo is not represented as standard int so you need to cast it to int for it to work in switch directly